app/assets/stylesheets/active_admin/components/_popovers.scss

```diff
@@ -0,0 +1,119 @@
+.popover {
+  display: inline-block;
+  position: absolute;
+  background-color: white;
+  padding: 2px;
+  box-shadow: rgba(0,0,0,0.4) 0 1px 3px, lighten($primary-color, 15%) 0px 1px 0px 0px inset;
```

Comment:
Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`